blankshield.js

```diff
@@ -81,19 +83,22 @@
    * @param {Event} e The click event for a given anchor
    */
   function clickListener(e) {
-    var target, href, usedModifier, child;
+    var target, targetName, href, usedModifier, child;
 
     // Use global event object for IE8 and below to get target
     e = e || window.event;
-    target = e.target || e.srcElement;
+    // Won't work for IE8 and below for cases when e.srcElement
+    // refers not to the anchor, but to the element inside it e.g. an image
+    target = e.currentTarget || e.srcElement;
```

Comment:
e.target won't work for links like
`<a href="https://mathiasbynens.be/demo/opener" target="_blank"><img src="image-placeholder.gif" /></a>`
as in that case e.target will return a ref to `<img>` element and its `href` property would be empty.

To get a ref to `<a>` element we need to use e.currentTarget

The problem with old IEs is they don't support e.currentTarget. There seems to be a way to emulate it (at least jQuery seems to do it), but I didn't dig deep into it.

So maybe my comment here is confusing.. Please feel free to change it to whatever you think is more suitable.